recipe/install-pkg.sh

```diff
@@ -61,8 +60,6 @@ do
     cp ${CHANGE}-${PKG_NAME}.sh ${PREFIX}/etc/conda/${CHANGE}.d/${CHANGE}-${PKG_NAME}.sh
 done
 
-CHOST=$CHOST_BASE$CL_VERSION
-
```

Comment:
Why remove this?

Reply:
Oversight while reverting https://github.com/conda-forge/clang-win-activation-feedstock/pull/18/commits/92002c36297480ee069ea34a6072fc8f575afc01, sorry